### PR TITLE
Added GitHub Action for Pytest

### PR DESCRIPTION
**Pull request by JDaniel41**
In this commit, I created a GitHub Action to run our `pytest` scripts for the lambda unit tests.

### CODE REVIEW

**Comment by JDaniel41** [comment]
Right now the action fails because there are no tests to run in this branch. However, when merged with the changes in #75 the action will pass since that version of the repository does include test cases.

**Review by dggorsen — COMMENTED** [review]
This is **awesome** love to see you running these tests via GitHub!  Will approve when the tests get approved

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            - name: Install Python 3
+              uses: actions/setup-python@v1
+              with:
+                  python-version: 3.9
+            - name: Install dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  pip install -r cdk/requirements.txt
+            - name: Test Lambdas
+              run: pytest cdk/visit/lambda_code
```
Comment (dggorsen) [inline .github/workflows/pytest.yml:24]: Will this run recursively?

**Comment by ophilli** [comment]
> Right now the action fails because there are no tests to run in this branch. However, when merged with the changes in #75 the action will pass since that version of the repository does include test cases.

Can you change the base branch of this PR so that it merges into #75 then?

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (JDaniel41) [reply .github/workflows/pytest.yml:24]: Yes, it will hit each file that has the `import pytest` line inside of `cdk/visit/lambda_code`

**Comment by JDaniel41** [comment]
Closing since these changes have now been folded into #75

**Pull request closed without merging**